packages/synapse-react-client/src/components/CreateTableViewWizard/TableNameForm.tsx

```diff
@@ -28,7 +30,8 @@ export default function TableNameForm(props: TableNameFormProps) {
         }}
         fullWidth
       />
-      {isInExperimentalMode && (
+      {(isInExperimentalMode ||
```

Comment:
With the suggested solution, we can also get rid of the `isInExperimentalMode` check, since that hook should do it for us